src/WireMock.Net/Owin/AspNetCoreSelfHost.cs

```diff
@@ -39,7 +39,7 @@ public AspNetCoreSelfHost([NotNull] WireMockMiddlewareOptions options, [NotNull]
 
             _logger = options.Logger ?? new WireMockConsoleLogger();
 
-            foreach (string uriPrefix in uriPrefixes)
+            foreach (string uriPrefix in uriPrefixes) // .Select(u => u.EndsWith("/") ? u : $"{u}/")) // Always append an / at the end.
```

Comment:
Is the code supposed to be commented out?

Reply:
Yes should be removed.